Formula/deno.rb

```diff
@@ -0,0 +1,35 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno.git",
+      :tag      => "v0.2.5",
+      :revision => "320577c131c8058ba0065f26119ce1ce18bc23d2"
+
+  bottle do
+  end
+
+  depends_on "ninja" => :build
+  depends_on "node" => :build
+  depends_on "python@2" => :build
+  depends_on "rust" => :build
+
+  def install
+    rm_rf ["node_modules", "buildtools", "build_overrides", "testing", "tools/clang"]
+    system "python", "tools/setup.py"
+    system "python", "tools/build.py", "-C", "target/release"
```

Comment:
Do we need to use python to manage the build script - why can't we not just use cargo instead ?

Reply:
"Cargo build should work (we test it) but we cut releases from the gn build "

We could change this to `cargo build` in the future if this changes in a future release.

Reply:
Go ahead and use cargo build if it helps. It works - we test it in CI.

Reply:
You will need python@2 installed even if using cargo.